syntax = "proto3";

package mdh.dvc.protobuf;

message OrderBookWide {
  string timestamp = 1;
  string symbol = 2;

 string bid_size1 = 6; // decimal
  string bid_price1 = 7; // decimal
  string ask_size1 = 8; // decimal
  string ask_price1 = 9; // decimal
  string bid_size2 = 10; // decimal
  string bid_price2 = 11; // decimal
  string ask_size2 = 12; // decimal
  string ask_price2 = 13; // decimal
  string bid_size3 = 14; // decimal
  string bid_price3 = 15; // decimal
  string ask_size3 = 16; // decimal
  string ask_price3 = 17; // decimal
  string bid_size4 = 18; // decimal
  string bid_price4 = 19; // decimal
  string ask_size4 = 20; // decimal
  string ask_price4 = 21; // decimal
  string bid_size5 = 22; // decimal
  string bid_price5 = 23; // decimal
  string ask_size5 = 24; // decimal
  string ask_price5 = 25; // decimal
  string bid_size6 = 26; // decimal
  string bid_price6 = 27; // decimal
  string ask_size6 = 28; // decimal
  string ask_price6 = 29; // decimal
  string bid_size7 = 30; // decimal
  string bid_price7 = 31; // decimal
  string ask_size7 = 32; // decimal
  string ask_price7 = 33; // decimal
  string bid_size8 = 34; // decimal
  string bid_price8 = 35; // decimal
  string ask_size8 = 36; // decimal
  string ask_price8 = 37; // decimal
  string bid_size9 = 38; // decimal
  string bid_price9 = 39; // decimal
  string ask_size9 = 40; // decimal
  string ask_price9 = 41; // decimal
  string bid_size10 = 42; // decimal
  string bid_price10 = 43; // decimal
  string ask_size10 = 44; // decimal
  string ask_price10 = 45; // decimal
  string bid_size11 = 46; // decimal
  string bid_price11 = 47; // decimal
  string ask_size11 = 48; // decimal
  string ask_price11 = 49; // decimal
  string bid_size12 = 50; // decimal
  string bid_price12 = 51; // decimal
  string ask_size12 = 52; // decimal
  string ask_price12 = 53; // decimal
  string bid_size13 = 54; // decimal
  string bid_price13 = 55; // decimal
  string ask_size13 = 56; // decimal
  string ask_price13 = 57; // decimal
  string bid_size14 = 58; // decimal
  string bid_price14 = 59; // decimal
  string ask_size14 = 60; // decimal
  string ask_price14 = 61; // decimal
  string bid_size15 = 62; // decimal
  string bid_price15 = 63; // decimal
  string ask_size15 = 64; // decimal
  string ask_price15 = 65; // decimal
  string bid_size16 = 66; // decimal
  string bid_price16 = 67; // decimal
  string ask_size16 = 68; // decimal
  string ask_price16 = 69; // decimal
  string bid_size17 = 70; // decimal
  string bid_price17 = 71; // decimal
  string ask_size17 = 72; // decimal
  string ask_price17 = 73; // decimal
  string bid_size18 = 74; // decimal
  string bid_price18 = 75; // decimal
  string ask_size18 = 76; // decimal
  string ask_price18 = 77; // decimal
  string bid_size19 = 78; // decimal
  string bid_price19 = 79; // decimal
  string ask_size19 = 80; // decimal
  string ask_price19 = 81; // decimal
  string bid_size20 = 82; // decimal
  string bid_price20 = 83; // decimal
  string ask_size20 = 84; // decimal
  string ask_price20 = 85; // decimal
  string bid_size21 = 86; // decimal
  string bid_price21 = 87; // decimal
  string ask_size21 = 88; // decimal
  string ask_price21 = 89; // decimal
  string bid_size22 = 90; // decimal
  string bid_price22 = 91; // decimal
  string ask_size22 = 92; // decimal
  string ask_price22 = 93; // decimal
  string bid_size23 = 94; // decimal
  string bid_price23 = 95; // decimal
  string ask_size23 = 96; // decimal
  string ask_price23 = 97; // decimal
  string bid_size24 = 98; // decimal
  string bid_price24 = 99; // decimal
  string ask_size24 = 100; // decimal
  string ask_price24 = 101; // decimal
  string bid_size25 = 102; // decimal
  string bid_price25 = 103; // decimal
  string ask_size25 = 104; // decimal
  string ask_price25 = 105; // decimal
  string bid_size26 = 106; // decimal
  string bid_price26 = 107; // decimal
  string ask_size26 = 108; // decimal
  string ask_price26 = 109; // decimal
  string bid_size27 = 110; // decimal
  string bid_price27 = 111; // decimal
  string ask_size27 = 112; // decimal
  string ask_price27 = 113; // decimal
  string bid_size28 = 114; // decimal
  string bid_price28 = 115; // decimal
  string ask_size28 = 116; // decimal
  string ask_price28 = 117; // decimal
  string bid_size29 = 118; // decimal
  string bid_price29 = 119; // decimal
  string ask_size29 = 120; // decimal
  string ask_price29 = 121; // decimal
  string bid_size30 = 122; // decimal
  string bid_price30 = 123; // decimal
  string ask_size30 = 124; // decimal
  string ask_price30 = 125; // decimal
  string bid_size31 = 126; // decimal
  string bid_price31 = 127; // decimal
  string ask_size31 = 128; // decimal
  string ask_price31 = 129; // decimal
  string bid_size32 = 130; // decimal
  string bid_price32 = 131; // decimal
  string ask_size32 = 132; // decimal
  string ask_price32 = 133; // decimal
  string bid_size33 = 134; // decimal
  string bid_price33 = 135; // decimal
  string ask_size33 = 136; // decimal
  string ask_price33 = 137; // decimal
  string bid_size34 = 138; // decimal
  string bid_price34 = 139; // decimal
  string ask_size34 = 140; // decimal
  string ask_price34 = 141; // decimal
  string bid_size35 = 142; // decimal
  string bid_price35 = 143; // decimal
  string ask_size35 = 144; // decimal
  string ask_price35 = 145; // decimal
  string bid_size36 = 146; // decimal
  string bid_price36 = 147; // decimal
  string ask_size36 = 148; // decimal
  string ask_price36 = 149; // decimal
  string bid_size37 = 150; // decimal
  string bid_price37 = 151; // decimal
  string ask_size37 = 152; // decimal
  string ask_price37 = 153; // decimal
  string bid_size38 = 154; // decimal
  string bid_price38 = 155; // decimal
  string ask_size38 = 156; // decimal
  string ask_price38 = 157; // decimal
  string bid_size39 = 158; // decimal
  string bid_price39 = 159; // decimal
  string ask_size39 = 160; // decimal
  string ask_price39 = 161; // decimal
  string bid_size40 = 162; // decimal
  string bid_price40 = 163; // decimal
  string ask_size40 = 164; // decimal
  string ask_price40 = 165; // decimal
  string bid_size41 = 166; // decimal
  string bid_price41 = 167; // decimal
  string ask_size41 = 168; // decimal
  string ask_price41 = 169; // decimal
  string bid_size42 = 170; // decimal
  string bid_price42 = 171; // decimal
  string ask_size42 = 172; // decimal
  string ask_price42 = 173; // decimal
  string bid_size43 = 174; // decimal
  string bid_price43 = 175; // decimal
  string ask_size43 = 176; // decimal
  string ask_price43 = 177; // decimal
  string bid_size44 = 178; // decimal
  string bid_price44 = 179; // decimal
  string ask_size44 = 180; // decimal
  string ask_price44 = 181; // decimal
  string bid_size45 = 182; // decimal
  string bid_price45 = 183; // decimal
  string ask_size45 = 184; // decimal
  string ask_price45 = 185; // decimal
  string bid_size46 = 186; // decimal
  string bid_price46 = 187; // decimal
  string ask_size46 = 188; // decimal
  string ask_price46 = 189; // decimal
  string bid_size47 = 190; // decimal
  string bid_price47 = 191; // decimal
  string ask_size47 = 192; // decimal
  string ask_price47 = 193; // decimal
  string bid_size48 = 194; // decimal
  string bid_price48 = 195; // decimal
  string ask_size48 = 196; // decimal
  string ask_price48 = 197; // decimal
  string bid_size49 = 198; // decimal
  string bid_price49 = 199; // decimal
  string ask_size49 = 200; // decimal
  string ask_price49 = 201; // decimal
  string bid_size50 = 202; // decimal
  string bid_price50 = 203; // decimal
  string ask_size50 = 204; // decimal
  string ask_price50 = 205; // decimal
}